.github/workflows/propagateUserHome.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Gradle init script to correct an unset 'user.name' Java system property resulting from the defect
+ * described in https://bugs.openjdk.java.net/browse/JDK-8193433.
+ */
+val userHome: String? = System.getProperty("user.home")
+if (userHome.isNullOrEmpty() || userHome.equals("?")) {
+    val home = System.getenv("HOME")
+    if (home.isNullOrEmpty()) {
+        throw IllegalStateException("environment variable \"HOME\" must be set when user.home='${userHome}'")
+    }
+
+    /*
+     * Replace Java's broken user.home value in both this JVM and for each test JVM.
+     */
+    println("Setting user.home=\"${home}\"")
```

Comment:
This should really go through a Gradle logger... I think at this layer you have to look it up through slf4j though.

Reply:
I just found out that `println` is sent to `logger.quiet` ...